cdc/generation.hh

```diff
@@ -64,21 +64,26 @@ namespace cdc {
 class stream_id final {
     bytes _value;
 public:
+    static constexpr uint8_t version_1 = 1;
```

Comment:
Silly question - why not start with 0? :-)

Reply:
Mainly because zero is the result of a = {}, so non-default values is nice.